### PR TITLE
Add additional EBS volume to control-plane nodes for `/var/log`

### PR DESCRIPTION
**Pull request by calvix**
towards https://github.com/giantswarm/giantswarm/issues/29019

- Add `log` volume to control-plane nodes.

default size 15GB equals to value that we use for vintage as a default

### CODE REVIEW

File: helm/cluster-aws/values.schema.json [hunk 1]
```diff
@@ -966,6 +995,11 @@
                                 "title": "Address range"
                             }
                         },
+                        "logVolumeSizeGB": {
+                            "type": "integer",
+                            "title": "Log volume size (GB)",
+                            "default": 15
```
Comment (whites11) [inline helm/cluster-aws/values.schema.json:1001]: 15 sounds like a very small default to me

Comment (calvix) [reply helm/cluster-aws/values.schema.json:1001]: thats what we do in vintage

Comment (calvix) [reply helm/cluster-aws/values.schema.json:1001]: So can you give me a better default? I am all ears I just don't have an idea

**Pull request merged**